Sources/MlemMiddleware/API Client/ApiClient.swift

```diff
@@ -215,6 +215,8 @@ extension ApiClient: CacheIdentifiable {
 
 extension ApiClient: ActorIdentifiable {
     public var actorId: URL { baseUrl }
+    
+    public var host: String? { actorId.host() }
```

Comment:
We use this exact line a couple other places, probably worth defining it as an extension of `ActorIdentifiable`